README.md

```diff
@@ -45,15 +51,18 @@ If you have a specific issue or bug, please file a Github issue. If you want to
 1. Restart it with `npm start`.
 
 #### Tips
+
 * I run the web server with `screen` to keep the web server running independently of the terminal session.
 * Ubuntu users may need to install `nodejs-legacy` (`sudo apt-get install nodejs-legacy`) for dependencies to install successfully. You can read more on [this Stack Overflow post](http://stackoverflow.com/questions/21168141/cannot-install-packages-using-node-package-manager-in-ubuntu).
 
 #### Local Development
+
 1. Run `npm install`.
 2. Run `npm run start:development:server` and `npm run start:development:client` in separate terminal instances.
   * `npm run start:development:server` uses [nodemon](https://github.com/remy/nodemon) to watch for changes to the server-side JavaScript.
   * `npm run start:development:client` watches for changes in the client-side source.
 3. Access the UI through the [WebpackDevServer](https://webpack.js.org/configuration/dev-server/). It expects to proxy requests to the Flood server you have running, defined in `config.js` as `floodServerProxy`.
+4. Build the documentation `./node_modules/.bin/jsdoc -c ./.jsdoc.json`.
```

Comment:
Would you mind moving this to an npm script and suggesting that here? Something like `npm run build-docs` would be great.

Reply:
What do you mean? Making a wrapper for the command and tell in the README to launch the wrapper  `npm run build-docs`?

Reply:
Yes, that's what I mean, sorry that was unclear.